pkg/chart/chart.go

```diff
@@ -129,6 +129,14 @@ func (ch *Chart) AppVersion() string {
 	return ch.Metadata.AppVersion
 }
 
+// ChartVersion returns the chartversion of the chart.
+func (ch *Chart) ChartVersion() string {
```

Comment:
This should be `Version` instead of `ChartVersion`.

Reply:
Thanks, I fixed the name!